UPGRADE-3.x.md

```diff
@@ -1,6 +1,56 @@
 UPGRADE 3.x
 ===========
 
+## Deprecated unspecified default_formatter configuration node
+
+The `default_formatter` configuration node will become required.
+
+```yaml
+sonata_formatter:
+    default_formatter: my_formatter
+```
+
+## Deprecated specifying a logger in the Sonata\FormatterBundle\Formatter\Pool constructor
+
+Providing a logger to a `Sonata\FormatterBundle\Formatter\Pool` instance is
+optional, and optional dependencies should be provided through a setter.
+
+Before:
+
+```
+use Sonata\FormatterBundle\Formatter\Pool;
+
+new Pool($myLogger, 'myFormatter');
+```
+
+After:
+
+```
+$pool = new Pool('myFormatter');
+$pool->setLogger($myLogger);
+```
+
+## Deprecated unspecified Sonata\FormatterBundle\Formatter\Pool defaultFormatter
```

Comment:
Deprecated unspecified default formatter in Sonata\FormatterBundle\Formatter\Pool constructor?